src/main/java/frc/robot/subsystems/ArmPivot.java

```diff
@@ -1,14 +1,49 @@
 package frc.robot.subsystems;
 
+import com.ctre.phoenix6.configs.TalonFXConfiguration;
+import com.ctre.phoenix6.configs.TalonFXConfigurator;
 import com.ctre.phoenix6.hardware.TalonFX;
+import com.ctre.phoenix6.signals.NeutralModeValue;
+import edu.wpi.first.wpilibj2.command.Command;
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
 import frc.robot.Hardware;
+import java.util.function.DoubleSupplier;
 
 public class ArmPivot extends SubsystemBase {
+  // Presets
+  public static int PRESET_L1 = 0;
+  public static int PRESET_L2_L3 = 35;
+  public static int PRESET_L4;
+  public static int HARDSTOP_HIGH = 90;
+  public static int HARDSTOP_LOW = 0;
+  public static double placeholderCoralWeight;
+  public static double placeholderAlgaeWeight;
 
-  private final TalonFX ring1;
+  // Remove once we implement PID speed
+  public static int placeholderPIDSpeed;
+
+  // TalonFX
+  private final TalonFX rings;
+  private TalonFXConfiguration configuration = new TalonFXConfiguration();
 
   public ArmPivot() {
-    ring1 = new TalonFX(Hardware.ARM_PIVOT_MOTOR_ID);
+    rings = new TalonFX(Hardware.ARM_PIVOT_MOTOR_ID);
+    factoryDefaults();
+  }
+
+  public Command gottaGo(DoubleSupplier iAmSpeed) {
+    return run(() -> rings.set(iAmSpeed.getAsDouble()));
+  }
+
+  public void moveArmCoral(int preset) {}
+
+  public void moveArmAlgae(int preset) {}
+
+  // TalonFX config
+  public void factoryDefaults() {
```

Comment:
Please add setting the current limits, like this does: https://github.com/robototes/Reefscape2025/blob/18c7f62ac296fb03407847f3696638de72777042/src/main/java/frc/robot/subsystems/ElevatorSubsystem.java#L116